.ci/conda.sh

```diff
@@ -1,16 +1,10 @@
 #!/usr/bin/env bash
+set -e -v
 
 NAME=$0
 COMMAND=$1
 MINICONDA="http://repo.continuum.io/miniconda/Miniconda-latest-Linux-x86_64.sh"
 
-function usage {
-    echo "usage: $NAME <command>"
-    echo
-    echo "  install  Install miniconda and a simple test environment"
-    exit 1
-}
```

Comment:
Yeah the main point of this was to give an error message if you called it with the wrong command.  Since in the new framework you can call any script with any command (and they give a warning message if it is a command they don't recognize), that didn't seem necessary anymore.

Reply:
Added a comment at the top explaining what the point of this script is

Reply:
For me part of having this message was also if you try to run it with no command, which is a common thing to forget when using it at the command line. I, for example, use `.ci/static.sh run` to run static checks locally so it's IMO a bit of a regression to have a worse error message if I forget to pass in `run`, but probably not worth blocking the merge...